charts/grafana/templates/grafana-deployment.yaml

```diff
@@ -35,6 +35,22 @@ spec:
       {{- if and (not .Values.backendSecretName) (not .Values.backendConnection) }}
       serviceAccountName: {{ template "grafana.bootstrapper.serviceaccount" . }}
       initContainers:
+        # TODO: add wait-for-db or similar to grafana container
+        # and remove the prisma dependency from this subchart
+        - name: prisma-wait-for-db
```

Comment:
Should this be `wait-for-db`? Prisma shouldn't be involved for Grafana.

Reply:
just because the prisma container is used. I can rename if it you want